protocols/virtualdefund/virtualdefund.go

```diff
@@ -71,6 +77,7 @@ type GetTwoPartyConsensusLedgerFunction func(counterparty types.Address) (ledger
 func NewObjective(request ObjectiveRequest,
 	preApprove bool,
 	myAddress types.Address,
+	largestPaymentAmount *big.Int,
```

Comment:
I think there's an underlying assumption here that by the time we've started `virtualDefund` we are done issuing or receiving vouchers. Since we set `latestVoucherAmount` at objective creation, the objective is going to ignore any new vouchers added to the `voucher manager`.

I think that's ok, but probably something to be aware of.

Reply:
I agree that it's OK and good to be aware.